### ISSUE TITLE
add dotnet 5 recipe for dunfell branch

### ISSUE THREAD

**Issue by karthago1** [DynamicDevices/meta-mono#97]
this is my first merge request, so I hope I did everything in the correct way.
this recipe is tested under dunfell branch and is copied from http://github.com/intel-iot-devkit/meta-iot-cloud/blob/master/recipes-devtools/dotnet/dotnet_5.0.300.bb

it will be nice if it will be included to the meta-mono (dunfell branch)

**Comment by ajlennon** [comment]
I'd like us to get this into the master branch if possible. Have you tried building against that?

**Comment by CTerasaGuF** [comment]
Not sure how this can be marked as "tested" at least for dunfell. The recipe obviously uses the more recent honister override syntax. See https://docs.yoctoproject.org/migration-guides/migration-3.4.html#override-syntax-changes
A plain old dunfell build does not work for me. But it should not be too hard to fix this.

**Comment by ajlennon** [comment]
These are the latest (building and tested) dunfell builds

https://github.com/DynamicDevices/meta-mono/actions/runs/1794025332

**Comment by CTerasaGuF** [comment]
Yes, I get the following error, but perhaps I am not using the latest poky dunfell version:
```ERROR: ParseError at /home/clemens.terasa/dunfell/sources/poky/meta-mono/recipes-mono/dotnet/dotnet_5.0.300.bb:15: unparsed line: 'RDEPENDS:${PN} = "    icu     libgssapi-krb5     lttng-ust     zlib "'```

I am preparing a patch for this that works for me and will propose a MR.

**Comment by karthago1** [comment]
yes you are right. I have no Idea why it is working with my dunfell branch. I will try to find out the reason
